Code/GraphMol/MolStandardize/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+
+rdkit_library(MolStandardize
+              MolStandardize.cpp
+	      Metal.cpp
+	      Normalize.cpp
+	      Validate.cpp
+	      Charge.cpp
+	      Tautomer.cpp
+        FragmentCatalog/FragmentCatalogEntry.cpp
+        FragmentCatalog/FragmentCatalogParams.cpp
+        FragmentCatalog/FragmentCatalogUtils.cpp
+        FragmentCatalog/FragmentRemover.cpp
+        AcidBaseCatalog/AcidBaseCatalogEntry.cpp
+        AcidBaseCatalog/AcidBaseCatalogParams.cpp
+        AcidBaseCatalog/AcidBaseCatalogUtils.cpp
+        TransformCatalog/TransformCatalogEntry.cpp
+        TransformCatalog/TransformCatalogParams.cpp
+        TransformCatalog/TransformCatalogUtils.cpp
+        TautomerCatalog/TautomerCatalogEntry.cpp
+        TautomerCatalog/TautomerCatalogParams.cpp
+        TautomerCatalog/TautomerCatalogUtils.cpp
+	      LINK_LIBRARIES ChemReactions ChemTransforms GraphMol RDGeneral )
+
+rdkit_headers(MolStandardize.h
+	      Metal.h
+	      Normalize.h
+	      Validate.h
+	      Charge.h
+	      Tautomer.h
+        FragmentCatalog/FragmentCatalogEntry.h
+        FragmentCatalog/FragmentCatalogParams.h
+        FragmentCatalog/FragmentCatalogUtils.h
+        FragmentCatalog/FragmentRemover.h
+        AcidBaseCatalog/AcidBaseCatalogEntry.h
+        AcidBaseCatalog/AcidBaseCatalogParams.h
+        AcidBaseCatalog/AcidBaseCatalogUtils.h
+        TransformCatalog/TransformCatalogEntry.h
+        TransformCatalog/TransformCatalogParams.h
+        TransformCatalog/TransformCatalogUtils.h
+        TautomerCatalog/TautomerCatalogEntry.h
+        TautomerCatalog/TautomerCatalogParams.h
+        TautomerCatalog/TautomerCatalogUtils.h
+	      DEST GraphMol/MolStandardize)
+
+add_subdirectory(Wrap)
+
+rdkit_test(molStandardizeTest test1.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch )
+rdkit_test(molNormalizeTest testNormalize.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch )
+rdkit_test(molValidateTest testValidate.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch FileParsers )
+rdkit_test(molChargeTest testCharge.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch FileParsers )
+rdkit_test(molTautomerTest testTautomer.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch FileParsers )
+rdkit_test(molStandardizeSmallTest test2.cpp LINK_LIBRARIES MolStandardize
+	SmilesParse RDGeneral GraphMol SubstructMatch)
+rdkit_test(molFragmentCatalogTest testFragmentCatalog.cpp LINK_LIBRARIES MolStandardize
+	Catalogs SmilesParse RDGeneral GraphMol SubstructMatch FileParsers
+	ChemTransforms Descriptors)
+
+find_package(Boost 1.56.0 COMPONENTS system iostreams REQUIRED)
```

Comment:
Might as well go ahead and comment out this section too if the only use of boost::iostreams is in `compareMolVSTest`